google/cloud/spanner/samples/samples.cc

```diff
@@ -439,6 +438,27 @@ void InstanceTestIamPermissionsCommand(std::vector<std::string> const& argv) {
   InstanceTestIamPermissions(std::move(client), argv[0], argv[1]);
 }
 
+google::cloud::spanner::Timestamp DatabaseNow(std::string const& project_id,
+                                              std::string const& instance_id,
+                                              std::string const& database_id) {
+  auto client = google::cloud::spanner::Client(
+      google::cloud::spanner::MakeConnection(google::cloud::spanner::Database(
```

Comment:
Consider refactoring to accept a `spanner::Client` parameter and reuse the parameter between example calls in `RunAll()`.

FYI, we generally use an existing client for these examples because it was simply too slow to create a client in each example (and that was when we had a fraction of the examples we have now).  Two or three calls as in this PR is not a big deal, but it can grow over time.

Reply:
Done, although it really only helps the `QueryWithTimestampParameter()` call.